ImageItem3D/ImageItem3D.js

```diff
@@ -87,11 +79,16 @@ const ImageItem3DBase = kind({
 		}, []);
 
 		const handleSelect = useCallback(() => { // eslint-disable-line react-hooks/rules-of-hooks
-			if (selected === index) {
-				setSelected(null);
+			if (setSelected) {
+				if (selected === index) {
+					setSelected(null);
+				} else {
+					setSelected(index);
+				}
 			} else {
-				setSelected(index);
+				return null;
 			}
+
```

Comment:
You can remove this empty line. (it's the same structure as for 'handlePointerDown' for example, and we don't have an empty line there - line 69)